src/HotelPosSystem.Tests/BookingTests.cs

```diff
@@ -0,0 +1,107 @@
+            Label emailAddress = GetElement("emailAddress2", bookingList).AsLabel();
+
+            Assert.Contains("robert.robertsson@example.com", emailAddress.Text);
+        }
+
+        [Fact]
+        public void PhoneNumberShouldBePresentInList() {
+            AutomationElement bookingList = GetElement(BookingListAutomationId, GetMainWindow());
+            Label phoneNumber = GetElement("phoneNumber3", bookingList).AsLabel();
+
+            Assert.Contains("070-1740640", phoneNumber.Text);
+        }
+
+        [Fact]
+        public void RoomNameShouldBePresentInList() {
+            AutomationElement bookingList = GetElement(BookingListAutomationId, GetMainWindow());
+            Label roomName = GetElement("roomName1", bookingList).AsLabel();
+
+            Assert.Contains("202", roomName.Text);
+        }
+
+        [Fact]
+        public void DatesShouldBePresentInList() {
+            AutomationElement bookingList = GetElement(BookingListAutomationId, GetMainWindow());
+            Label startEndDatesLabel = GetElement("dates2", bookingList).AsLabel();
+
+            Assert.Contains("2023-12-27", startEndDatesLabel.Text);
+            Assert.Contains("2024-01-03", startEndDatesLabel.Text);
+        }
+
+        [Fact]
+        public void CheckedInStatusShouldBeCorrect() {
+            AutomationElement bookingList = GetElement(BookingListAutomationId, GetMainWindow());
+            CheckBox checkedInCheckBox = GetElement("checkedIn1", bookingList).AsCheckBox();
+            CheckBox notCheckedInCheckBox = GetElement("checkedIn2", bookingList).AsCheckBox();
+
+            Assert.True(checkedInCheckBox.IsChecked);
+            Assert.False(notCheckedInCheckBox.IsChecked);
+        }
+
+        [Fact]
+        public void PaidStatusShouldBeCorrect() {
+            AutomationElement bookingList = GetElement(BookingListAutomationId, GetMainWindow());
+            CheckBox paidCheckBox = GetElement("paidFor2", bookingList).AsCheckBox();
+            CheckBox notPaidCheckBox = GetElement("paidFor1", bookingList).AsCheckBox();
+
+            Assert.True(paidCheckBox.IsChecked);
+            Assert.False(notPaidCheckBox.IsChecked);
+        }
+
+        [Fact]
+        public void CommentShouldBePresentInList() {
+            AutomationElement bookingList = GetElement(BookingListAutomationId, GetMainWindow());
+            Label commentLabel = GetElement("comment1", bookingList).AsLabel();
+
+            Assert.Contains("Cleaning crew one hour late", commentLabel.Text);
+        }
+
+        [Fact]
+        public void CommentShouldNotBePresentInList() {
+            AutomationElement bookingList = GetElement(BookingListAutomationId, GetMainWindow());
+            AutomationElement? commentLabel = GetElement("comment2", bookingList);
+
+            Assert.Null(commentLabel);
+        }
+
+        private AutomationElement GetElement(string automationId, AutomationElement parent) {
```

Comment:
```suggestion
        private AutomationElement GetElement(string automationId, AutomationElement parent) {
```
Lägg till en "documentation comment" här som förklarar att automationId är det som kallas "name" i winforms